src/entity.c

```diff
@@ -10,6 +11,8 @@ entity* CreateEntity(SDL_Renderer *ren, int x, int y, char *imagePath) {
     entity *e = malloc(sizeof(entity));
     e->x = x;
     e->y = y;
+    e->w = w;
+    e->h = h;
```

Comment:
Saving width and height to entity properties means that in future, when changing the sprite, we'll need to remember to change `w` and `h` properties. Is this better than just querying the texture on collision detection function?